packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/declaration-order/output.js

```diff
@@ -1,9 +1,11 @@
 var C = function C() {
   "use strict";
 
+  var _this = this;
+
   babelHelpers.classCallCheck(this, C);
-  this.y = babelHelpers.classPrivateFieldLooseBase(this, _x)[_x];
-  Object.defineProperty(this, _x, {
+  _this.y = babelHelpers.classPrivateFieldLooseBase(_this, _x)[_x];
```

Comment:
(Stylistic opinion)
I highly prefer `this` over `_this` when possible. I find it clearer, and it is gzipped as fine as `_this`.